packages/nextjs/components/Footer.tsx

```diff
@@ -42,18 +45,19 @@ export const Footer = () => {
               </a>
             </div>
             <span>·</span>
-            <div>
+            <div className="flex justify-center items-center gap-1">
               <p className="m-0 text-center">
-                Built with <HeartIcon className="inline-block h-4 w-4" /> at 🏰{" "}
-                <a
-                  href="https://buidlguidl.com/"
-                  target="_blank"
-                  rel="noreferrer"
-                  className="underline underline-offset-2"
-                >
-                  BuidlGuidl
-                </a>
+                Built with <HeartIcon className="inline-block h-4 w-4" /> at
               </p>
+              <div className="relative w-20 h-4 pb-5">
+                <a href="https://buidlguidl.com/" target="_blank" rel="noreferrer">
+                  {isDarkMode ? (
+                    <Image alt="SE2 logo" className="pb-1 cursor-pointer" fill src="/bg-logo-light.svg" />
```

Comment:
Change svg to tsx component, see for example `/components/example-ui/assets`. Use https://react-svgr.com/playground/?exportType=named to convert attributes to react-compatible. Change `fill="#color` to `fill="currentColor"`, so it will take color of the parent. So you'll have one component and don't need `useDarkMode` anymore and also use whatever color you want.

Also I don't like the space between logo and text, but I think we need to ask designers to change that

Reply:
I think it's better to take only icon without text here

Reply:
If we want to keep the text as well, I can quickly increase the spacing between the logo and the text.   Here is a quick go at it:  https://www.figma.com/file/AD1r8jWVZrCke80K0kgfH8/Untitled?type=design&node-id=0%3A1&mode=design&t=ufPl843Dm68mBax2-1

Reply:
Can also use a separate file for the logo and for the text so then y'all can work your magic.

Reply:
Thank you for your help @rin-st @ZakGriffith .

@rin-st , I did your suggested SVG-magic, renamed the file and changed to only use one file, neat!

I don't agree that there should be more spacing of the logo. But I tried some different approaches and attach their printscreens below. Let me know what you think.

Only icon and with underlined text.
![Screenshot from 2023-10-01 10-30-46](https://github.com/scaffold-eth/scaffold-eth-2/assets/6756284/e5451d23-716d-4f17-ae10-60702a8ead95)

Icon + text with extra spacing from v2 given by @ZakGriffith [here](https://www.figma.com/file/AD1r8jWVZrCke80K0kgfH8/Untitled?type=design&node-id=0%3A1&mode=design&t=ufPl843Dm68mBax2-1).
![Screenshot from 2023-10-01 10-26-26](https://github.com/scaffold-eth/scaffold-eth-2/assets/6756284/408d8b35-117e-4597-9eb1-0b26a12ae5fa)

Icon + text from [original from here](https://www.figma.com/file/jPaXdmHNfbTSwBWwENr6Iq/BG-Media-Kit?type=design&node-id=1-29&mode=design&t=kUu7PIZFibkQ8mrz-0).
![Screenshot from 2023-10-01 10-23-30](https://github.com/scaffold-eth/scaffold-eth-2/assets/6756284/174c4385-a189-4717-9149-3fb633262e12)

Reply:
Voting for first. Best spacing and it's clear that you can click it. 2-3 are good too, but I think with spacing like in 1 it could be better.

> Can also use a separate file for the logo and for the text so then y'all can work your magic.

I think we don't need it because devs can break base proportions between Icon and text. Probably better to have different versions of (Icon_spacing_text) if proportions or spacings are different.

But here since we use it just on footer we need just one version or Icon separately if we decide to go with first version.

Reply:
OK, I updated code with only icon (first pic). I can just revert it if we need to change a again.

Reply:
Alright, I don't know who is allowed to resolve this thread. But the current PR is with nbr 1. 

@ZakGriffith @rin-st

Reply:
Looking good!